pyaedt/modeler/cad/Modeler.py

```diff
@@ -1840,6 +1884,139 @@ def set_working_coordinate_system(self, name):
             )
         return True
 
+    @pyaedt_function_handler()
+    def invert_cs(self, coordinate_system, to_global=False):
+        """
+        Get the origin and quaternion of the inverse transformation of that which defines the
+        input coordinate system. By defining a coordinate system with this information, the
+        Reference CS of the input one is obtained.
```

Comment:
The one-line description of this method should be 100 characters or less. This description is not clear. Based on my limited understanding, how about:

        Get the origin and quaternion of the inverse transformation defining the input coordinate system.

        By defining a coordinate system with this information, the reference coordinate system of the input
        one is obtained.  
        
        (Note that I have no idea what "one" is in "the input one is obtained".

Reply:
The purpose of the function is simple. If I define a reference system with a translation and a rotation, the function provides inverse translation and inverse rotation.

What do you think of:
`Get the inverse translation and the conjugate quaternion of the transformation that defines the input coordinate system.

By defining a coordinate system with this information, the reference coordinate system of the input coordinate system is obtained. `